src/shared/models/game/GameSetup.es6

```diff
@@ -0,0 +1,83 @@
+// const BaseError = require('../error/BaseError')
+const Player = require('./Player')
+const Market = require('./Market')
+const Tile = require ('./Tile')
+// const GameState = require ('./GameState')
+const _ = require('underscore')
+
+
+const GameSetup = {
```

Comment:
Yeah so I mentioned this in chat, but GameState is the representation of state that we'll be keeping and using throughout the game.

GameSetup is just a mixin for setting up the initial game state before any turns have been made to keep things neat / separate out the diff concerns.